superduperdb/cdc/app.py

```diff
@@ -3,7 +3,7 @@
 from superduperdb.components.listener import Listener
 from superduperdb.server import app as superduperapp
 
-assert isinstance(CFG.cluster.cdc, str)
+assert isinstance(CFG.cluster.cdc, str), "Missing SUPERDUPERDB_CLUSTER_CDC"
```

Comment:
`SUPERDUPERDB_CLUSTER_CDC` is a way to set, CFG.cluster.cdc 

so assertion error should be

`cluster.cdc should be set with a valid uri`
something on the above terms.